src/content/basics/cluster-size-autoscaling/index.md

```diff
@@ -5,6 +5,7 @@ date = "2020-02-17"
 weight = 120
 type = "page"
 categories = ["basics"]
+last-review-date = "2020-03-31"
```

Comment:
oh, just noticed this.. is this a new thing we are doing? I think this field doesn't really get used programatically in any way, is it just to keep a note for yourself when you last saw it?

Any reason we dont can't `date` above for that?

Reply:
It is indeed a new thing and I am adding it for internal purposes. Sometimes a doc doesn't need to be edited, but it does require a review every once in a while.